doc/api/next_api_changes/behavior/24131-OG.rst

```diff
@@ -0,0 +1,6 @@
+Passing None as ``save_count``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+... to `.FuncAnimation` no longer limits the number of frames to 100. Make
+sure that it either can be inferred from *frames* or provide an integer
+*save_count*.
```

Comment:
```suggestion
``FuncAnimation(save_count=None)``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Passing ``save_count=None`` to `.FuncAnimation` no longer limits the number
of frames to 100. Make``` sure that it either can be inferred from *frames*
or provide an integer *save_count*.
```

FuncAnimation should be in the title to provide the context